workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-cron/source-formatter-suppressions.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+
+<suppressions>
+	<source-check>
+		<suppress checks="IllegalImportsCheck" files="src/main/java/com/liferay/sample/SampleCommandLineRunner\.java" />
+	</source-check>
+</suppressions>
```

Comment:
Can this go in the root level suppressions file? That way the suppressions file doesn't get included in the published artifact. Thoughts?